server/templates/student/course/_assigntable.html

```diff
@@ -9,7 +9,7 @@
           {% if scores %}<th class="col-scores">Scores</th> {% endif %}
           <th class="col-status">Status</th>
           <th class="col-due-date">Due Date</th>
-          <th class="col-group">Group</th>
+          <th class="col-group hidden-md">Group</th>
```

Comment:
Or make group the second column for both tables and move scores to the last column?